src/components/Gallery/GridImage.tsx

```diff
@@ -46,6 +54,26 @@ export const GridImage: FunctionComponent<GridImageComponentProps> = ({
       />
     );
   };
+
+  const GridVideoAsset = () => {
+    return imageDidError ? (
+      <VideoAssetSVG />
+    ) : (
+      <Imgix
+        src={VideoThumbnail() || ''}
+        width={140}
+        height={125}
+        imgixParams={{
+          auto: 'format',
+          fit: 'crop',
+          crop: 'entropy',
```

Comment:
Did we try `crop: 'edges'` instead here? Entropy can yield weird results with text.

Reply:
Discussed offline.